src/utils.ts

```diff
@@ -162,7 +162,13 @@ export const ocHide = () => {
     oc.hide();
 };
 
-export const randomId = () => Math.floor(Date.now() + Math.random() * 100);
+export const randomId = (index: number | null) => {
+    if (index) {
```

Comment:
If index is 0 it index will be false-like. Do an explicit null check here ( index === null )